.github/workflows/forwardport.yml

```diff
@@ -5,36 +5,8 @@ on:
       - v/*
 jobs:
   forwardport:
-    strategy:
-      matrix:
-        branch: ['main']
-    runs-on: ubuntu-latest
-    steps:
-    
-      - name: checkout
-        uses: actions/checkout@v4
-        with:
-         fetch-depth: 0
-
-      - name: Set up git config
-        run: |
-          git config user.name "GitHub Actions Bot"
-          git config user.email "<>"
-          
-      - name: Check PR for forwardport label
-        id: check_pr_labels
-        uses: shioyang/check-pr-labels-on-push-action@v1.0.12
-        with:
-         github-token: ${{ secrets.GITHUB_TOKEN }}
-         labels: '["forwardport to snapshot"]'
-        
-      - name: See result
-        run: echo "${{ steps.check_pr_labels.outputs.result }}"
-
-      - name: Checkout branch and cherry-pick
-        if: ${{ steps.check_pr_labels.outputs.result == 'true' }}
-        run: |
-          git fetch
-          git checkout ${{ matrix.branch }}
-          git cherry-pick -x --strategy=recursive -X theirs $GITHUB_SHA
-          git push
+    uses: ./.github/workflows/backport-workflow.yml
+    with:
+      label: '["forwardport to snapshot"]'
```

Comment:
I'm on the fence on this one, too - I think it might be clearer as `main` vs `SNAPSHOT`, but no-ones complained (although it's [not terribly well used](https://github.com/hazelcast/hz-docs/pulls?q=is%3Apr+label%3A%22forwardport+to+snapshot%22)).

Reply:
Sure. lets keep it.